generators/app/templates/express-api/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM alpine:3.3
```

Comment:
I wouldn't say that it is better necessarily, just you know exactly what you are getting